doc/distrib/NodeHelpFiles/_review/Revit.Elements.ScheduleOnSheet.ScheduleView.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`ScheduleOnSheet.ScheduleView` returns the owner schedule view used to generate the instance on the sheet.
+
+In the example below, the schedule instances are returned for the selected sheet. The revision schedules are then filtered out with `List.FilterByBoolMask`. Additionally, the owner view for the schedule instance is retrieved with `ScheduleOnSheet.ScheduleView`.
```

Comment:
If the example is changed, note to update this section of the In Depth description.